portfolio/src/main/webapp/index.html

```diff
@@ -43,17 +43,10 @@ <h1>Ikechukwu Daniel Adebi - Portfolio</h1>
        <p>Click the button below to get a random fact about me.</p>
       <button onclick="addRandomGreeting()">Random Fact Button</button> 
       <div id="greeting-container"></div>
-
+      <br />
       <div id="comments-section"></div>
       <form action="/data" method="POST">
       <p>Max Comment Count:</p>
-      <!-- <select type="number" name="max-comments" id="max-comments" onchange="getComments()">
-        <option value="1">1</option>
-        <option value="2">2</option>
-        <option value="3">3</option>
-        <option value="4">4</option>
-        <option value="5">5</option>
-      </select> -->
       <input type="number" name="max-comments" id="max-comments" onchange="getComments()" min="1" value="3" required=true> 
```

Comment:
change this so that the attributes line up on a multi-line like

```
<input type="number"
       name="max-comments"
       id="max-comments"
       ...
       required=true>
```

Reply:
Done!